test/test_helper.exs

```diff
@@ -1 +1,6 @@
 ExUnit.start()
+
+alias EctoPlusOne.Repo
+
+{:ok, _} = Ecto.Adapters.Postgres.ensure_all_started(Repo, :temporary)
```

Comment:
just curious why is this necessary here and not on other test_helpers, for example not in naboo...

Reply:
this is not necessary it turns out. removing